packages/execution-environments/src/common/endowments/index.ts

```diff
@@ -50,12 +50,17 @@ export function createEndowments(
           // `attenuatedEndowments` object, but will only be passed on to the snap
           // if explicitly listed among its endowment.
           // This may not have an actual use case, but, safety first.
-          Object.assign(
-            attenuatedEndowments,
-            // We just confirmed that endowmentFactories has the specified key.
+
+          // We just confirmed that endowmentFactories has the specified key.
+
+          // @ts-expect-error Todo
```

Comment:
We should fix these comments. They're weirdly spaced and have a "todo".

Reply:
Done!